src/sentry/interfaces/stacktrace.py

```diff
@@ -146,8 +159,10 @@ def validate_bool(value, required=True):
 class Frame(Interface):
     @classmethod
     def to_python(cls, data):
-        abs_path = data.get('abs_path')
-        filename = data.get('filename')
+        abs_path = maybe_force_text(data.get('abs_path'))
```

Comment:
it might make more since to also just assert on some of these

basically we dont want people submitting non strings for things that should be strings and now that we error people should be able to figure it out

(we can also add better errors for interface validation vs just the 'bad data')